File: api/app/models/keg.js
var moment = require('moment')
var Request = require('tedious').Request;
var TYPES = require('tedious').TYPES;
var Connection = require('tedious').Connection;
var env = require('dotenv').load();

function keg(name, tapId, volumeInGallons, brewery, beerType, abv, ibu, beerDescription, untappdId, imagePath){
    this.name = name;
    this.tapId = tapId;
    this.volumeInGallons = volumeInGallons;

    this.brewery = brewery;
    this.beerType = beerType;
    this.abv = abv;
    this.ibu = ibu;
    this.beerDescription = beerDescription;
    this.untappdId = untappdId;
    this.imagePath = imagePath;
    var date = new Date();
    this.installDate = moment.utc().format("YYYY-MM-DD hh:mm:ss");
}

keg.prototype = {

}

File: mobileapp/CommonLib/Keg.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Elixir.Common
{
    public struct Keg
    {
        public int KegId { get; set; }
        public string Name { get; set; }
        public string Brewery { get; set; }
        public string BeerType { get; set; }
        public int ABV { get; set; }
        public int IBU { get; set; }
        public string BeerDescription { get; set; }
        public int? UntappdId { get; set; }
        public string imagePath { get; set; }
    }
}
